calebwagner/Martins-Aquarium
# I changed the fish header title

Martin wanted the fish title changed to be more informative.

## Steps to Test

1. pull this branch
2. open in chrome
3. verify the the fish title has changed in the first section
4. make sure the h1 title in the first section works and is changed
